src/modules/auth/resources/ResourcesDeletedDetailContainer.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import {
+	Container, Row, Col, Card,
+	CardHeader, CardBody, Button,
+	CardFooter, ButtonGroup
+} from 'reactstrap';
+
+import { DateTime, ButtonWithAuthz } from 'asab-webui';
+
+const ResourceDetailContainer = (props) =>  {
+	const { t } = useTranslation();
+	const SeaCatAuthAPI = props.app.axiosCreate('seacat_auth');
+	const [resource, setResource] = useState(null);
+	const [editMode, setEditMode] = useState(false);
+	const { resource_id } = props.match.params;
+
+	const resources = useSelector(state => state.auth?.resources);
+
+	const getResourceDetail = async (res) => {
+		try {
+			let response = await SeaCatAuthAPI.get(`resource/${res}`);
+			setResource(response.data);
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesDeletedDetailContainer|Can't fetch resource details")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+
+	useEffect(() => {
+		getResourceDetail(resource_id);
+	}, []);
+
+	if (!resource) return null;
+
+	// Set delete resource dialog
+	const confirmForm = (type) => {
+		var r = confirm(t(`ResourcesDeletedDetailContainer|Do you really want to ${type === "delete" ? 'hard-delete' : 'retrieve'} this resource`));
+		if (r == true) {
+			if (type === "delete") {
+				hardDelete()
+			} else {
+				retrieveResource();
+			}
+		}
+	}
+
+	// Retrieve the resource
+	const retrieveResource = async () => {
+		try {
+			let response = await SeaCatAuthAPI.post(`/resource/${resource_id}`, {});
+			if (response.data.result !== "OK") {
+				throw new Error(t("ResourcesDeletedDetailContainer|Failed to retrieve resource"));
+			}
+			props.app.addAlert("success", t("ResourcesDeletedDetailContainer|Resource retrieved successfuly"));
+			props.history.push("/auth/resources");
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesDeletedDetailContainer|Failed to retrieve resource")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+
+	const hardDelete = async () => {
+		try {
+			let response = await SeaCatAuthAPI.delete(`/resource/${resource_id}`, { params: { hard_delete: true } });
+			if (response.data.result !== "OK") {
+				throw new Error(t("ResourcesDeletedDetailContainer|Failed to hard-delete this resource"));
+			}
+			props.app.addAlert("success", t("ResourcesDeletedDetailContainer|Resource was successfully hard-deleted"));
+			props.history.push("/auth/resources");
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesDeletedDetailContainer|Failed to hard-delete this resource")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+
+
+	return (
+		<Container>
+			<Row className="mb-4 justify-content-md-center">
```

Comment:
@petrKavulok What if you will try to use grid areas? In this single Card page it might be a good exercise

Reply:
I'll create a ticket and put it to backlog

Reply:
@petrKavulok my note about grid areas is still valid